web/src/components/dashboard/SectorModal.test.tsx

```diff
@@ -65,19 +92,34 @@ describe("<SectorModal />", () => {
     ).toBeInTheDocument();
   });
 
-  it("calls onContinue prop on successful submit", () => {
+  it("calls onContinue prop on successful submit", async () => {
     const business = generateBusiness({
       profileData: generateProfileData({
         sectorId: undefined,
         industryId: "generic",
       }),
     });
+    const userData = generateUserDataForBusiness(business);
+
+    mockApiResponse(userData, {
+      profileData: {
+        ...business.profileData,
+        municipality: undefined,
+      },
+      taxFilingData: generateTaxFilingData({
+        state: "SUCCESS",
+        businessName: business.profileData.businessName,
+        errorField: undefined,
+      }),
+    });
```

Comment:
[pebble] I don't think we actually need to be mocking the API response here. The `<SectorModal>` is used inside `<SidebarCardFundingNudge>`, but this component's logic isn't dependent on the API.

Let me know if I'm missing something